com.unity.multiplayer.mlapi/Runtime/Hashing/XXHash/XXHash.cs

```diff
@@ -5,6 +5,7 @@
 
 // @mfatihmar (Unity): Modified for Unity support
 
+using System.Text;
```

Comment:
FYI — changes in this file are coming from PR #688 and after merging #688 prior to this PR, this change will not show up, so please do NOT review changes in this file to save yourself some time :)